### PR TITLE
Update configure-endpoints-gp.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the doc as environments that have NTLM disabled will have problems using the NetBIOS name.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5787

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5785

File: microsoft-365/security/defender-endpoint/configure-endpoints-gp.md [hunk 1]
```diff
@@ -67,9 +67,11 @@ Check out the [PDF](https://github.com/MicrosoftDocs/microsoft-365-docs/raw/publ
 
 7. Select **Run whether user is logged on or not** and check the **Run with highest privileges** check box.
 
-8. Go to the **Actions** tab and click **New...** Ensure that **Start a program** is selected in the **Action** field. Enter the NetBIOS path of the shared  *WindowsDefenderATPOnboardingScript.cmd* file.
+8. In the Name field, type an appropriate name for the scheduled task (e.g., Defender for Endpoint Deployment).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-gp.md]:
```suggestion
8. In the Name field, type an appropriate name for the scheduled task (for example, Defender for Endpoint Deployment).
```

File: microsoft-365/security/defender-endpoint/configure-endpoints-gp.md [hunk 2]
```diff
@@ -67,9 +67,11 @@ Check out the [PDF](https://github.com/MicrosoftDocs/microsoft-365-docs/raw/publ
...
 
-9. Click **OK** and close any open GPMC windows.
+9. Go to the **Actions** tab and click **New...** Ensure that **Start a program** is selected in the **Action** field. Enter the UNC path, using the file server's fully qualified domain name (FQDN), of the shared *WindowsDefenderATPOnboardingScript.cmd* file.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-gp.md]:
```suggestion
9. Go to the **Actions** tab and select **New...** Ensure that **Start a program** is selected in the **Action** field. Enter the UNC path, using the file server's fully qualified domain name (FQDN), of the shared *WindowsDefenderATPOnboardingScript.cmd* file.
```

File: microsoft-365/security/defender-endpoint/configure-endpoints-gp.md [hunk 3]
```diff
@@ -67,9 +67,11 @@ Check out the [PDF](https://github.com/MicrosoftDocs/microsoft-365-docs/raw/publ
...
+
+10. Click **OK** and close any open GPMC windows.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-gp.md]:
```suggestion
10. Select **OK** and close any open GPMC windows.
```

File: microsoft-365/security/defender-endpoint/configure-endpoints-gp.md [hunk 4]
```diff
@@ -212,9 +214,11 @@ For security reasons, the package used to Offboard devices will expire 30 days a
 
 7. Select **Run whether user is logged on or not** and check the **Run with highest privileges** check-box.
 
-8. Go to the **Actions** tab and click **New...**. Ensure that **Start a program** is selected in the **Action** field. Enter the NetBIOS path of the shared *WindowsDefenderATPOffboardingScript_valid_until_YYYY-MM-DD.cmd* file.
+8. In the Name field, type an appropriate name for the scheduled task (e.g., Defender for Endpoint Deployment).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-gp.md]:
```suggestion
8. In the Name field, type an appropriate name for the scheduled task (for example, Defender for Endpoint Deployment).
```

File: microsoft-365/security/defender-endpoint/configure-endpoints-gp.md [hunk 5]
```diff
@@ -212,9 +214,11 @@ For security reasons, the package used to Offboard devices will expire 30 days a
...
+
+9. Go to the **Actions** tab and click **New...**. Ensure that **Start a program** is selected in the **Action** field. Enter the UNC path, using the file server's fully qualified domain name (FQDN), of the shared *WindowsDefenderATPOffboardingScript_valid_until_YYYY-MM-DD.cmd* file.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-gp.md]:
```suggestion
9. Go to the **Actions** tab and select **New...**. Ensure that **Start a program** is selected in the **Action** field. Enter the UNC path, using the file server's fully qualified domain name (FQDN), of the shared *WindowsDefenderATPOffboardingScript_valid_until_YYYY-MM-DD.cmd* file.
```

File: microsoft-365/security/defender-endpoint/configure-endpoints-gp.md [hunk 6]
```diff
@@ -212,9 +214,11 @@ For security reasons, the package used to Offboard devices will expire 30 days a
...
 
-9. Click **OK** and close any open GPMC windows.
+10. Click **OK** and close any open GPMC windows.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-gp.md]:
```suggestion
10. Select **OK** and close any open GPMC windows.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @msbemba, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**